Explorer/Assets/Scripts/SceneRunner/SceneFacade.cs

```diff
@@ -86,9 +87,23 @@ public async UniTask StartUpdateLoopAsync(int targetFPS, CancellationToken ct)
 
             SetTargetFPS(targetFPS);
 
-            // Start the scene
+            try
+            {
+                // Start the scene
+                await runtimeInstance.StartScene();
+            }
+            catch (OperationCanceledException) { return; }
```

Comment:
I believe we don't need to catch `OperationCanceledException`: it will be never fired. We don't have logic for cancellation from JS